source/service_test.go

```diff
@@ -113,29 +118,50 @@ func testServiceSourceEndpoints(t *testing.T) {
 			"",
 			"testing",
 			"foo",
+			v1.ServiceTypeLoadBalancer,
 			"",
 			"",
 			map[string]string{},
 			map[string]string{
 				hostnameAnnotationKey: "foo.example.org.",
 			},
+			"",
 			[]string{"1.2.3.4"},
 			[]*endpoint.Endpoint{
 				{DNSName: "foo.example.org", Target: "1.2.3.4"},
 			},
 			false,
 		},
 		{
+			"annotated ClusterIp aren't processed without explicit authorization",
+			"",
+			"testing",
+			"foo",
+			v1.ServiceTypeClusterIP,
+			"",
+			"",
+			map[string]string{},
+			map[string]string{
+				hostnameAnnotationKey: "foo.example.org.",
+			},
+			"1.2.3.4",
```

Comment:
Awesome :) I was referring to the test coverage for "if the ClusterIP service with "None" IP field is found it should be ignored" case